source/fundamentals/qe.txt

```diff
@@ -0,0 +1,140 @@
+====================
+{+qe+}
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to install and use **{+qe+}**
+in the MongoDB Node.js driver.
+
+{+qe+} allows you to encrypt specific data fields within a document with
+your MongoDB client application before sending the data to the server.
+Starting in MongoDB 6.0 Enterprise and Atlas, you can perform this
+client-side encryption automatically.
+
+With {+qe+}, your client application encrypts fields client-side. {+qe+}
+is useful for situations in which applications must guarantee that
+unauthorized parties, including server administrators, cannot read the
+encrypted data.
+
+This guide is a quick introduction to {+qe+} using the Node.js driver.
+For in-depth information on how {+qe+} works, see the
+:ref:`{+qe+} reference <qe-features>` documentation.
+
+Installation
+------------
+
+To get started with {+qe+} in your client application, you need:
+
+- the MongoDB Node.js driver
+- `mongodb-client-encryption <https://www.npmjs.com/package/mongodb-client-encryption>`__ version ``2.2.0-alpha.1`` or later
+- ``crypt_shared`` or ``mongocryptd`` if using automatic encryption
+
+``mongodb-client-encryption``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``mongodb-client-encryption`` module is the official client
+encryption module for the MongoDB Node.js driver. It contains bindings
+to communicate with the native library that manages the encryption.
+
+Add it to your project using ``npm``:
+
+.. code-block:: sh
+   :copyable: true
+
+   npm install mongodb-client-encryption@2.2.0-alpha.1 --save
+
+``crypt_shared``
+~~~~~~~~~~~~~~~~
+
+``crypt_shared`` is automatically used by the package, and is used
+for automatic encryption. It is a dynamically-loaded library providing
+query-analysis, and replaces ``mongocryptd``.
+
+For more detailed information on ``crypt_shared``, see the
+:ref:`crypt_shared reference documentation <qe-crypt_shared>`.
+
+``mongocryptd``
+~~~~~~~~~~~~~~~
+
+``mongocryptd`` is launched automatically by the package, and is used for
+automatic encryption. ``mongocryptd`` communicates with
+``mongodb-client-encryption`` to automatically encrypt the information
+specified by a user-provided
+:manual:`JSON Schema </reference/security-client-side-automatic-json-schema/>`.
```

Comment:
This might be a bit confusing, since QE doesn’t do JSON Schema anymore (at least not explicitly)?